src/plugins/intel_cpu/src/memory_desc/cpu_memory_desc_utils.cpp

```diff
@@ -135,5 +136,11 @@ Shape MemoryDescUtils::makeDummyShape(const Shape &shape, const VectorDims& dumm
     }
     return Shape(dummyDims);
 }
+
+std::string MemoryDescUtils::computeWeightsStringHash(const MemoryCPtr memory, const std::shared_ptr<DnnlMemoryDesc> dstDesc) {
+    const auto desc_hash = dnnl::impl::primitive_hashing::get_md_hash(*dstDesc->getDnnlDesc().get());
+    return std::to_string(desc_hash) + "_" + std::to_string(memory->getSize()) + "_" +
```

Comment:
Do we need to keep size as part of hash given full memory desc is now its part?

Reply:
You are right, the memory size is already hashed in `desc_hash`. I removed a separate hash computation for `memory->getSize()`